schema.json

```diff
@@ -1378,7 +1542,7 @@
     "Certificate": {
       "description": "",
       "type": "object",
-      "required": ["CertificateLanguages", "CommercialTransaction", "ProductDescription", "Inspection", "Validation"],
+      "required": ["CertificateLanguages", "CommercialTransaction", "ProductDescription", "Validation"],
```

Comment:
We have learned in the CoA project that there are certificates having no inspections at all, something to be done in a future release. If we want to keep it consistent then I would break things now.
I made the changes in the template locally (fixing another issue in course). If you agree we continue with that and I open an issue in schema-tools.

Reply:
Ok, this also has to be fixed in the related schema-tools PR to provide a reliable PDF generator, I left a review for Eamon.

Reply:
Thank you.